### ISSUE TITLE
Unable to use tire:import:all switching from 0.5.8 to 0.6.0

### ISSUE THREAD

**Issue by vnorguet** [karmi/retire#806]
I use ActiveSupport::Concerns to plug my indexation code into models.
In 0.5.8 : rake import:all works ok (indexes all my models)
In 0.6.0 : rake import:all is broken
`[IMPORT] Loading models from: /home/vnorguet/workspace/gportal/app/models
rake aborted!
uninitialized constant SkinTag::SearcheableSkinTag
/home/vnorguet/workspace/gportal/app/models/skin_tag.rb:6:in `<class:SkinTag>'
/home/vnorguet/workspace/gportal/app/models/skin_tag.rb:1:in `<top (required)>'
/home/vnorguet/.rvm/gems/ruby-1.9.3-p429@gnet/gems/tire-0.6.0/lib/tire/tasks.rb:124:in `block (4 levels) in <top (required)>'
/home/vnorguet/.rvm/gems/ruby-1.9.3-p429@gnet/gems/tire-0.6.0/lib/tire/tasks.rb:123:in `each'
/home/vnorguet/.rvm/gems/ruby-1.9.3-p429@gnet/gems/tire-0.6.0/lib/tire/tasks.rb:123:in `block (3 levels) in <top (required)>'
/home/vnorguet/workspace/gportal/lib/tasks/es.rake:4:in `block (2 levels) in <top (required)>'
/home/vnorguet/.rvm/gems/ruby-1.9.3-p429@gnet/bin/ruby_noexec_wrapper:14:in `eval'
/home/vnorguet/.rvm/gems/ruby-1.9.3-p429@gnet/bin/ruby_noexec_wrapper:14:in `<main>'
Tasks: TOP => es:reindex
(See full trace by running task with --trace)`

Here's the tree stucture of the project :
-app
 |- controllers
 |- mixins
 |- models

Here is the header of the Concern :

``` ruby
module SearcheableSkinTag
  extend ActiveSupport::Concern
  included do
    include Tire::Model::Search
    include Tire::Model::Callbacks
    include Searcheable
```

and here the headers of the model :

``` ruby
class SkinTag < ActiveRecord::Base
  attr_accessible nil
  include SearcheableSkinTag
```

Is there something I forgot to update?

**Comment by vnorguet** [comment]
Sorry, I've posted a bit fast. If another one asked the question, cf:
https://github.com/karmi/tire/issues/708

rake environment tire:import:model CLASS='MyModel'

**Comment by karmi** [comment]
Are you calling it like this `rake environment tire:import:all`?

**Comment by karmi** [comment]
Can you provide more detailed description of what exactly is not working as expected?

**Comment by vnorguet** [comment]
Hello,

Sorry for late I was out of internet for few days.
Here's how I called the task in 0.5.8

``` ruby
rake tire:import:all
```

Here how I've fixed the problem for version 0.6.0 :

``` ruby
rake environment tire:import:all
```

For me, this issue can be closed, since it was a bad practice not to use "environment" in the rake task.

**Comment by karmi** [comment]
OK, we briefly introduced autoloading environment, removed it in favor of the dependency being handled Rake itself.
